packages/example-app-base/src/components/EditSection/EditSection.types.ts

```diff
@@ -5,39 +5,32 @@ import { IRefObject } from 'office-ui-fabric-react/lib/Utilities';
 export interface IEditSection {}
 
 export interface IEditSectionProps extends React.HTMLAttributes<EditSection> {
-  /**
-   * Optional callback to access the IPersona interface. Use this instead of ref for accessing
-   * the public methods and properties of the component.
-   */
+  /** @deprecated Not used */
   componentRef?: IRefObject<IEditSection>;
 
   /** Optional className for the root element. */
   className?: string;
 
   /**
-   * The name of the Component
+   * The name of the component
    */
   title: string;
 
   /**
-   * The section id of the page.
+   * The section id this button is used to edit.
```

Comment:
nit: We have `ID` in caps later for the `readableSection` prop, maybe we could also make this read `The section ID ...`?

Reply:
Changed it to uppercase. Though I suspect it's often written in lowercase in our doc comments as a reference to the `id` HTML attribute (so I'm not sure which way is "right").